worldedit/manipulations.lua

```diff
@@ -38,6 +38,29 @@ function worldedit.set(pos1, pos2, node_names)
 	return worldedit.volume(pos1, pos2)
 end
 
+--- Sets param2 of a region.
+-- @param pos1
+-- @param pos2
+-- @param param2 Value of param2 to set
+-- @return The number of nodes set.
+function worldedit.set_param2(pos1, pos2, param2)
+	pos1, pos2 = worldedit.sort_pos(pos1, pos2)
+
+	local manip, area = mh.init(pos1, pos2)
+	local param2_data = {}
+
+	-- Set param2 for every node
+	for i in area:iterp(pos1, pos2) do
+		param2_data[i] = param2
+	end
+
+	-- Update map
+	manip:set_param2_data(param2_data)
+	manip:write_to_map()
+	manip:update_map()

```

Comment:
update_map is deprecated and does nothing now

Reply:
Well, then `manip_helper` needs updating, too because the function is used there, too.

Reply:
l think adding code which does nothing is less desirable than not removing code which does nothing.